scvi/inference/fish.py

```diff
@@ -5,17 +5,24 @@
 from scvi.inference import Posterior
 from scvi.inference import Trainer
 from scvi.models.classifier import Classifier
-from scvi.models.log_likelihood import compute_log_likelihood
+from scvi.models.log_likelihood import compute_reconstruction_error, compute_elbo
 
 
 class FishPosterior(Posterior):
 
     @torch.no_grad()
-    def ll(self, verbose=False):
-        ll = compute_log_likelihood(self.model, self, mode="smFISH")
+    def elbo(self, verbose=False):
+        elbo = compute_elbo(self.model, self, mode="smFISH")
         if verbose:
-            print("LL Fish: %.4f" % ll)
-        return ll
+            print("ELBO Fish: %.4f" % elbo)
+        return elbo
+
+    @torch.no_grad()
+    def reconstruction_error(self, verbose=False):
+        reconstruction_error = compute_reconstruction_error(self.model, self, mode="smFISH")
+        if verbose:
```

Comment:
Are these `verbose`  flags useful? There's something to be said for just adding in breakpoints for debugging. But I'm just asking.

Reply:
If they aren't useful, it'd be nice to just directly call `compute_reconstruction_error` rather than adding another method that does almost nothing except call `compute_reconstruction_error`.

Reply:
> Are these `verbose` flags useful? There's something to be said for just adding in breakpoints for debugging. But I'm just asking.

Not really, especially especially when we'll have completely switched to logging



> If they aren't useful, it'd be nice to just directly call `compute_reconstruction_error` rather than adding another method that does almost nothing except call `compute_reconstruction_error`.

I did not want to break the calls that had `verbose` param in them, especially in the notebooks since it would more or less require a full re-run to have to corresponding output.